tutorials/plot_sensors_time_frequency.py

```diff
@@ -70,12 +72,103 @@
 ax.fill_between(freqs, psds_mean - psds_std, psds_mean + psds_std,
-                color='k', alpha=.5)
+                color='k', alpha=.15, lw=0)
 ax.set(title='Multitaper PSD (gradiometers)', xlabel='Frequency',
        ylabel='Power Spectral Density (dB)')
 plt.show()
 
 ###############################################################################
+# To see the difference between :func:`psd_multitaper` and :func:`psd_welch`
+# - we are going to compare PSDs obtained with each function.
+ch_index = epochs.ch_names.index('MEG 2333')
+
+psds_m, freqs_m = psd_multitaper(epochs, picks=[ch_index],
+                                 fmin=2, fmax=17, n_jobs=1)
+psds_w, freqs_w = psd_welch(epochs, n_fft=epochs.info['sfreq'] * 2,
+                            picks=[ch_index], fmin=2, fmax=17, n_jobs=1)
+
+# drop channel dimension and average epochs
+psd_m = psds_m.squeeze(axis=1).mean(axis=0)
+psd_w = psds_w.squeeze(axis=1).mean(axis=0)
+
+# normalize multitaper and welch to put them on the same scale
+psd_m /= psd_m.sum() / len(freqs_m)
+psd_w /= psd_w.sum() / len(freqs_w)
+
+# plot
+f, ax = plt.subplots()
+ax.plot(freqs_m, psd_m, label='multitaper', lw=2)
+ax.plot(freqs_w, psd_w, label='welch', lw=2)
+ax.set(title='Multitaper vs Welch PSD (gradiometers)', xlabel='Frequency',
+       ylabel='Power Spectral Density')
+ax.legend()
+plt.show()
+
+###############################################################################
+# We can observe that multitaper estimation yields smoother spectrum - the
+# amount of smoothing can be controlled with ``bandwidth`` argument:
+bandwidths = np.arange(1, 7)
+colors = plt.cm.viridis(bandwidths / 7)
+
+fig, ax = plt.subplots()
+
+for bnd_idx, bnd in enumerate(bandwidths):
+    psd, freqs = psd_multitaper(epochs, picks=[ch_index],
+                                fmin=2, fmax=17, bandwidth=bnd)
+    ax.plot(freqs, psd[:, 0].mean(axis=0), label='bandwidth={}'.format(bnd),
+            color=colors[bnd_idx], lw=2)
+
+ax.set(title='Multitaper with different bandwidth', xlabel='Frequency',
+       ylabel='Power Spectral Density')
+ax.legend(loc='best')
+plt.show()
+
+###############################################################################
+# While in multitaper the averaging is done across independent realizations of
+# the signal (using slepian tapers), welch method averages across time
+# segments of the signal (often overlapping). Instead of averaging the windows
+# you can choose a different reduction by specifying ``combine``:
+welch_args = dict(picks=[ch_index], n_fft=epochs.info['sfreq'],
+                  n_overlap=int(epochs.info['sfreq'] / 2), fmin=2, fmax=25)
+psds_w, freqs = psd_welch(epochs, combine='mean', **welch_args)
+psds_w_trimmed, _ = psd_welch(epochs, combine=0.15, **welch_args)
+psds_w_median, _ = psd_welch(epochs, combine='median', **welch_args)
+
+psd_w = psds_w[:, 0].mean(axis=0)
+psd_w_trim = psds_w_trimmed[:, 0].mean(axis=0)
+psd_w_med = psds_w_median[:, 0].mean(axis=0)
```

Comment:
Hm, yes I agree that it is kind of weird. :) I can change it to median or I'll change that example later so that psd_welch is presented on raw data (as you suggested in other comments).